uaclient/cli.py

```diff
@@ -263,7 +263,8 @@ def action_attach(args, cfg):
     if not contract_token:
         print('No valid contract token available')
         return 1
-    if not contract.request_updated_contract(cfg, contract_token):
+    if not contract.request_updated_contract(
+            cfg, contract_token, allow_enable=True):
```

Comment:
ua enable is the only path that allows UAEntitlement.enable() call if the procesed deltas contain an enableByDefault: True obligation.